Packs/CrowdStrikeFalcon/Playbooks/playbook-CrowdStrike_Falcon_Malware_-_Incident_Enrichment.yml

```diff
@@ -428,7 +428,7 @@ tasks:
     task:
       id: 76cc6bfa-107f-4440-85c0-066f67bede92
       version: -1
-      name: Check if we have more than 1 item
+      name: Is there only 1 endpoint?
       description: Checks whether given entries returned an error. Use ${lastCompletedTaskEntries} to check the previous task entries. If an array is provided, will return 'yes' if one of the entries returns an error.
```

Comment:
I liked better the task name before

Reply:
But it was incorrect :) - in the previous task, we are trying to set with an array, and only after the condition we are setting one element...
I can suggest another option `Check if there is only 1 item`

Reply:
Change only the 1 -> `one`